### PR TITLE
Make changes in package.json and .gitignore to publish on unpkg

### PR DESCRIPTION
**Pull request by ThomasG77**
It was several commits due to @mourner feedbacks (restoring main block + changing scripts)
We've rebased it.

### CODE REVIEW

**Comment by ThomasG77** [comment]
If my PR OK, is it possible to tag master to make a first release on both npm and unpk.org?

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ tmp
 *.log
 trees*.json
 demo/supercluster.js
+dist/
```
Comment (mourner) [inline .gitignore]: Please add a new line here.

Comment (ThomasG77) [inline .gitignore]: Done in https://github.com/mapbox/supercluster/pull/22/commits/81e1b598b6d3176c9f44c38c030b4ef51ea6769c

**Comment by mourner** [comment]
@ThomasG77 published as v2.2.0.

**Comment by ThomasG77** [comment]
Thanks for the publication

Maybe adding a link to the file in the README could help.
Would you prefer to reference
https://npmcdn.com/supercluster@2.2.0/dist/supercluster.min.js
or
https://unpkg.com/supercluster@2.2.0/dist/supercluster.min.js
as they are equivalent.

**Comment by mourner** [comment]
@ThomasG77 not equivalent — npmcdn name is getting deprecated in favor of unpkg.

**Pull request merged**